packages/merge-styles/src/shadowConfig.ts

```diff
@@ -1,9 +1,10 @@
-export type ShadowConfig = {
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export interface ShadowConfig {
```

Comment:
interface is faster to parse, also provides "most of the time" nicer inferrence UX